apps/laboratory/src/utils/ProfilesUtil.ts

```diff
@@ -0,0 +1,235 @@
+import {
```

Comment:
are these utils intended to be implemented by developers?

Reply:
Nope it was for testing purposes around profiles.